syntax = "proto3";

message LoginRequest {
  string username = 1;
  string password = 2;
  string captcha = 3;
}
message LoginResponse {
  string response = 1;
}
message ActivitiyRequest {
  string username = 1;
}
message ActivityResponse {
  string responseJson = 1;
}
message UserInfoRequest {
  string username = 1;
}
message UserinfoResponse {
  string responseJson = 1;
}
service ZhihuService {
  rpc Login(LoginRequest) returns (LoginResponse) {}
  rpc GetActivity(ActivitiyRequest) returns (ActivityResponse)  {}
  rpc GetUserInfo(UserInfoRequest) returns (UserinfoResponse) {}
}